motoman_mpx3500_support/urdf/mpx3500_macro.xacro

```diff
@@ -0,0 +1,170 @@
+<?xml version="1.0" ?>
+<robot xmlns:xacro="http://ros.org/wiki/xacro">
+  <xacro:macro name="motoman_mpx3500" params="prefix">
+
+    <!-- Properties -->
+    <material name="yaskawa_blue">
+          <color rgba="0.12941 0.14902 0.74902 1" />
+    </material>
+    <material name="yaskawa_white">
+          <color rgba="0.95 0.95 0.95 1" />
+    </material>
+    <material name="yaskawa_silver">
+          <color rgba="0.75294 0.75294 0.75294 1" />
+    </material>
+
+    <!-- link list -->
+    <link name="${prefix}base_link">
+        <visual>
+            <geometry>
+                <mesh filename="package://motoman_mpx3500_support/meshes/mpx3500/visual/base_link.stl"/>
+            </geometry>
+            <material name="yaskawa_blue"/>
+        </visual>
+        <collision>
+            <geometry>
+                <mesh filename="package://motoman_mpx3500_support/meshes/mpx3500/collision/base_link.stl"/>
+            </geometry>
+        </collision>
+    </link>
+    <link name="${prefix}link_1_s">
+        <visual>
+            <geometry>
+                <mesh filename="package://motoman_mpx3500_support/meshes/mpx3500/visual/link_1_s.stl"/>
```

Comment:
Do `link`s actually have a name?

`S` is the joint name afaik, but this would just be "the first link", would it not?

If they don't have names, it would be better to not repeat the joint identifier here in the filenames and only use `link_1.stl` etc.

Reply:
It makes reference to the Yaskawa standard link identification: SLURBT
Stands for Swivel, Lower, Upper, Roll, Bent, Twist.
All the models have been defined like this so far, I'm not sure that we want to start changing that now...

Reply:
All the `joint`s have been named that way, it's not necessarily true for the `link`s.

I know what the letters stand for, I was just curious whether they refer to the `joint`s or the `link`s.

Reply:
@gavanderhoorn Thanks for looking it over.  I have a customer that will be using that model, so I'll wait for there feedback that everything is moving well before merging.